### ISSUE TITLE
`Set Library Search Order` doesn't work if there are two matches and one is from standard libraries

### ISSUE THREAD

**Issue by jaasal** [robotframework/robotframework#4516]
Versions: Robot Framework 6.0, Python 3.8.10, Windows 10
Prerequisites: robotframework-sshlibrary installed

Execute the following test:
```
*** Settings ***
Library  SSHLibrary
Library  OperatingSystem

*** Test Cases ***
Test Library Search Order
  Set Library Search Order  OperatingSystem  SSHLibrary  
  Get File  ./foo.txt
```

Expected: Get File -keyword from OperatingSystem is executed
Actual: Warning about duplicate keywords is shown, Get File -keyword from SSHLibrary is executed

**Comment by pekkaklarck** [comment]
Did this work before RF 6.0?

**Comment by pekkaklarck** [comment]
Ok. I think what caused that. Needs to be fixed in 6.0.1.
